### ISSUE TITLE
Broken URL in Custom Templates Docs

### ISSUE THREAD

**Issue by dt-snyder** [forcedotcom/salesforcedx-vscode#3627]
It's hard to follow the instructions to clone a repo no one has access to. Please fix this. 

Reference Line: https://github.com/forcedotcom/salesforcedx-vscode/blob/fce42fc0ea4fd65f4bfa7836576ba0ff286fb408/docs/_articles/en/user-guide/byotemplate.md?plain=1#L9

Broken URL: https://github.com/forcedotcom/salesforcedx-templates/tree/develop/packages/templates/src/templates

Public Location of Docs: https://developer.salesforce.com/tools/vscode/en/user-guide/byotemplate

**Comment by randi274** [comment]
Thanks for catching this @dt-snyder! We changed out branching strategy for that repo, and this was an unforeseen consequence of that.
